common/djangoapps/student/models.py

```diff
@@ -998,7 +998,9 @@ class CourseEnrollment(models.Model):
     history = HistoricalRecords()
 
     # cache key format e.g enrollment.<username>.<course_key>.mode = 'honor'
-    COURSE_ENROLLMENT_CACHE_KEY = u"enrollment.{}.{}.mode"
+    COURSE_ENROLLMENT_CACHE_KEY = u"enrollment.{}.{}.mode"  # TODO Can this be removed?  It doesn't seem to be used.
```

Comment:
It's used here: https://github.com/edx/edx-platform/pull/15036/files#diff-e2ffffdf6d67c9e0dee5bc10a67eb8bfL1682